plantcv/plantcv/visualize/pseudocolor.py

```diff
@@ -6,20 +6,23 @@
 from matplotlib import pyplot as plt
 from plantcv.plantcv import params
 from plantcv.plantcv import fatal_error
+from plantcv.plantcv.apply_mask import apply_mask
 
 
-def pseudocolor(gray_img, obj=None, mask=None, cmap=None, background="image", min_value=0, max_value=255,
-                axes=True, colorbar=True, obj_padding="auto", title=None):
+def pseudocolor(gray_img, obj=None, mask=None, bad_mask=None, cmap=None, background="image", bad_color="red",
```

Comment:
@DannieSheng, not sure we had a consensus, but what do you think about moving `bad_mask` and `bad_color` to the end of the arguments (this matches the documentation actually) to make this a non-breaking update?

Reply:
Sure!